src/Table.js

```diff
@@ -44,17 +47,26 @@ class Table extends React.Component {
 
     const classes = {
       ...getClassSet(bsProps),
-      [prefix(bsProps, 'responsive')]: responsive,
-      [prefix(bsProps, 'inverse')]: inverse,
+      [prefix(bsProps, 'dark')]: dark,
       [prefix(bsProps, 'striped')]: striped,
       [prefix(bsProps, 'bordered')]: bordered,
-      [prefix(bsProps, 'condensed')]: condensed,
+      [prefix(bsProps, 'sm')]: sm,
```

Comment:
this feels like a weird choice by the bootstrap team since there is only one size. I'm guessing they were leaving space for other sizes? in which case this might make more sense as a `size` prop, like in Button.

Reply:
I think similarly with `dark` that should become `variant='dark'` like Button maybe

Reply:
In that case, striped and bordered should also be variants right? 

Also, apologies for the delay.